deployments/ecmps/release-v2.0-fix/Sprint24/6726/camdecmpswks/delete_calculated_qa_data_from_workspace.sql

```diff
@@ -0,0 +1,192 @@
+-- FUNCTION: camdecmpswks.delete_calculated_qa_data_from_workspace()
+
+DROP FUNCTION IF EXISTS camdecmpswks.delete_calculated_qa_data_from_workspace(character varying[]) CASCADE;
```

Comment:
Should this also drop the previous function signature that is no longer in use? I don't see that in the Dev database anymore, so I'm assuming it was done separately.
```suggestion
DROP FUNCTION IF EXISTS camdecmpswks.delete_calculated_qa_data_from_workspace() CASCADE;
DROP FUNCTION IF EXISTS camdecmpswks.delete_calculated_qa_data_from_workspace(character varying[]) CASCADE;
```

Reply:
Yep 👍  You are correct. I have committed your suggestion.